wsimod/nodes/nodes.py

```diff
@@ -87,6 +87,17 @@ def apply_overrides(self, overrides: Dict[str, Any] = {}) -> None:
         Args:
             overrides (dict, optional): Dictionary of overrides. Defaults to {}.
         """
+        # overrides data_input_dict
+        from wsimod.orchestration.model import read_csv
+
+        content = overrides.pop("data_input_dict", self.data_input_dict)
+        if isinstance(content, str):
+            self.data_input_dict = read_csv(content)
+        elif not content:
+            pass
+        else:
+            print("ERROR: not recognised format for data_input_dict")
```

Comment:
This should be an exception, so the program halts.
```suggestion
            raise RuntimeError("Not recognised format for data_input_dict")
```
If, for any reason, you do not want to consider it a catastrophic failure and the program can keep going, then you should use a logger with either `error` or `warning`:

```python 
from logging import getLogger

getLogger().error("Not recognised format for data_input_dict")
```